src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -812,7 +812,7 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                      nEdges = nEdgesPtr
                      nEdges = nEdgesArray( edgeHaloComputeCounter )
 
-                     !$omp do schedule(runtime) private(cell1, cell2, CoriolisTerm, i, eoe)
+                     !$omp do schedule(runtime)
                      do iEdge = 1, nEdges
 
                         temp_mask = edgeMask(1, iEdge)
```

Comment:
Here is an example: `temp_mask` was added to the region, but not declared in the private clause. So `private(cell1, cell2, CoriolisTerm, i, eoe)` is now outdated/incomplete.